.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ executors:
     working_directory: /mnt/ramdisk
     environment:
       - BASH_ENV: /home/circleci/.bashrc
-      - ES_JAVA_OPTS: -Xms2g -Xmx3g
+      - ES_JAVA_OPTS: -Xms2g -Xmx2g
```

Comment:
Hmm, I did just change this for circle. Guess it's fine as long as indexing tests are still consistent.

Reply:
They should be equal: https://www.elastic.co/guide/en/elasticsearch/reference/7.x/heap-size.html#heap-size

Reply:
Have read that, and wouldn't recommend it for production. There did however seem to be some advantage for circle in particular by not assigning too much memory at the beginning but allowing JVM to grow if needed. In any case not a big deal if indexing tests performing better.